### PR TITLE
Elvis/card design

### PR DESCRIPTION
**Pull request by emarc0314**
Bug Fixes

## Overview

Fixed the onboarding screen, implemented cell to be released 

## Changes Made

omitted the constraints for capacityStatusLabel in GymCellFooter.swift

</details>

### CODE REVIEW

File: Uplift/Views/Footers/GymCellFooter.swift [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+                return ClientStrings.CommonStrings.closed
+            } else {
+                return ClientStrings.Home.gymDetailCellOpensAt + gymHoursTomorrow.openTime.getStringOfDatetime(format: format)
+            }
+        } else if !isOpen {
+            format = gymHoursToday.openTime.getHourFormat()
+            return ClientStrings.Home.gymDetailCellOpensAt + (gymHoursToday.openTime.getStringOfDatetime(format: format))
+        } else {
+            format = gymHoursToday.closeTime.getHourFormat()
+            let openTime = gymHoursToday.openTime.getStringOfDatetime(format: format)
```
Comment (aldenlamp) [inline Uplift/Views/Footers/GymCellFooter.swift]: I this line necessary?

File: Uplift/Views/Footers/GymCellFooter.swift [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+            let closeTime = gymHoursToday.closeTime.getStringOfDatetime(format: format)
+
+            return ClientStrings.Home.gymDetailCellClosesAt + closeTime
+        }
```
Comment (aldenlamp) [inline Uplift/Views/Footers/GymCellFooter.swift]: This seems a little unintuitive to me/hear to hear. I'll differ judgment to @ReadeP if it actually needs to/should be reworked.

At the least, I think you should comment this to make it very clear what is happening.

File: Uplift/Views/Footers/GymCellFooter.swift [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+        statusLabel.text = gym.isOpen ? ClientStrings.CommonStrings.open : ClientStrings.CommonStrings.closed
+
+        // Set gym hours
+        hoursLabel.text = getHoursString(from: gym)
+
+        capacityStatusLabel.text = "Cramped"
+        capacityCountLabel.text = "120/140"
+    }
+
+    private func getHoursString(from gym: Gym) -> String {
```
Comment (aldenlamp) [inline Uplift/Views/Footers/GymCellFooter.swift]: I would move this function to the `gym` class.

**Review by aldenlamp — CHANGES_REQUESTED** [review]

File: Uplift/Controllers/HomeViewController+Extensions.swift [hunk 4]
```diff
@@ -72,7 +73,7 @@ extension HomeViewController: UICollectionViewDataSource {
         case .checkIns:
             headerView.configure(title: sections[indexPath.section].rawValue, buttonTitle: editButtonTitle, completion: pushHabitOnboarding)
         case .myGyms:
-            headerView.configure(title: sections[indexPath.section].rawValue, buttonTitle: "", completion: pushGymOnboarding)
+            headerView.configure(title: sections[indexPath.section].rawValue, buttonTitle: nil, completion: pushGymOnboarding)
         case .todaysClasses:
             headerView.configure(title: sections[indexPath.section].rawValue, buttonTitle: "", completion: viewTodaysClasses)
```
[the remark this answers is not in the document]
Comment (ReadeP) [reply Uplift/Controllers/HomeViewController+Extensions.swift:78]: +1

**Review by ReadeP — APPROVED** [review]
LGTM! Just let Alden also review it as well before merging

**Review by aldenlamp — APPROVED** [review]
lgtm

**Pull request merged**